extra/cherry.py

```diff
@@ -119,7 +119,7 @@ def riski_unop(op):
   elif op == UnaryOps.LOG:
     regfile[Reg.MATMUL_OUTPUT] = np.log(regfile[Reg.MATMUL_INPUT])
   elif op == UnaryOps.EXP:
-    regfile[Reg.MATMUL_OUTPUT] = np.exp(regfile[Reg.MATMUL_INPUT])
+    regfile[Reg.MATMUL_OUTPUT] = np.asarray([np.exp(regfile[Reg.MATMUL_INPUT][i,:]) for i in range(regfile[Reg.MATMUL_INPUT].shape[0])])
```

Comment:
Wait huh, how does this do anything?

Reply:
Jeez how noob of me. I was on b0 from different testing and forgot.

So I cant figure out any other way of avoiding the overflow limit other than hard clip or a soft clip (which seems redundant due to overhead).

to overcome this would there be some sort of normalization to add in training on the 0-255 pixel values?

There's only one input array that causes the overflow.
828th EXP op.
row index: 9
column index: 9
value: 93.67906188964844